py3status/modules/volume_status.py

```diff
@@ -53,7 +58,8 @@
 ```
 
 Requires:
-    alsa-utils: (tested with alsa-utils 1.0.29-1)
+    alsa-utils: alsa backend (tested with alsa-utils 1.0.29-1)
+    pamixer: pulseaudio backend
```

Comment:
One or the other, which is why I added the `XXX backend`. I'll make it more explicit.